### PR TITLE
implement own tabbar, proptypes fixes

### LINKED ISSUE #364: short TabBars still get full 1920px width
The effect of this is that on short bars such as Distributions, no text is visible to the user without scrolling right blindly.

### LINKED ISSUE #385: TabBar elements aren't right-clickable
This breaks the ability to open in new tab.  We should make these <Link> elements.

### PR DESCRIPTION
**Pull request by howardchung**
fixes #385 
fixes #364 

Reviewers:
@blukai 

@blukai maybe you could help me style this better?

### CODE REVIEW

**Comment by blukai** [comment]
Sure. 🦄 Can I push commits right here?

**Comment by howardchung** [comment]
Yeah go ahead

**Comment by blukai** [comment]
image

File: src/components/Match/Match.jsx [hunk 1]
```diff
@@ -38,6 +39,7 @@ class RequestLayer extends React.Component {
           <TabBar
             info={info}
             tabs={matchPages(matchId, match)}
+            mediaQClass={styles.tabBar}
```
Comment (howardchung) [inline src/components/Match/Match.jsx]: why does the Match TabBar need this while the others don't?

Comment (blukai) [inline src/components/Match/Match.jsx]: Match tabBar have a lot of tabs, the others don't.

**Pull request merged**